README.md

```diff
@@ -271,14 +271,12 @@ render() {
 `StyleGuideProvider`'s props are used to set the page head properties using [Helmet](https://github.com/nfl/react-helmet).
 
 
-## Release Process
+## Contributing
 
-This package follows [semantic versioning](http://semver.org). In the `master` branch, run `npm version` with the appropriate version bump:
+This package follows [semantic versioning](http://semver.org). New versions are published automatically from [Travis CI](https://travis-ci.org) using [semantic-release](https://github.com/semantic-release/semantic-release).
 
-```bash
-$ npm version major
-$ npm version minor
-$ npm version patch
-```
+In order to automatically increment version numbers correctly, commit messages must follow the [AngularJS commit message convention](https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#-git-commit-guidelines). To make this easier, you can use [commitizen](https://github.com/commitizen/cz-cli) to guide you through the commit process by running `npm run commit` from the command line.
+
```

Comment:
Do we like badges?
[![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
`[![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)`